integration_tests/17_docker_benchmark_storm_ok.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Test for https://github.com/testground/testground/pull/1393
+# (Benchmark storm test)
+my_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$my_dir/header.sh"
+
+testground plan import --from ./plans --name testground
+
+pushd $TEMPDIR
+
+testground healthcheck --runner local:docker --fix
+
+# Run test
+testground run single \
+    --plan=testground/benchmarks \
+    --testcase=storm \
+    --builder=docker:go \
+    --runner=local:docker \
+    --instances=2 \
+    --wait | tee run.out
+
+assert_run_outcome_is ./run.out "success"
+
+popd
```

Comment:
@brdji Is there a reason not to run this in CI currently? I don't see it in https://github.com/testground/testground/blob/master/Makefile#L59